modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -142,6 +161,8 @@ private static int setCompletedFlag(int sequenceNumber) {
         return 0;
     }
 
+
```

Comment:
Remove unnecessary new lines. Check in all places

Reply:
https://github.com/wso2/product-sp/pull/855/commits/3248156c1c90f9d63629b154810672b57e2b3fb9